presto-docs/src/main/sphinx/sql/explain-analyze.rst

```diff
@@ -0,0 +1,52 @@
+===============
+EXPLAIN ANALYZE
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    EXPLAIN ANALYZE statement
+
+Description
+-----------
+
+Execute the statement, and show the distributed execution plan of the statement
+along with the cost of each operation.
+
+.. note::
+
+    The stats may not be 100% accurate, so cost for some part of the query may be unknown
+
+Examples
+--------
+
+.. code-block:: none
+
+    presto:sf1> EXPLAIN ANALYZE SELECT count(*), clerk FROM orders GROUP BY clerk;
```

Comment:
For consistency we should use the same driving example as for EXPLAIN (https://prestodb.io/docs/current/sql/explain.html) i.e. "SELECT regionkey, count(*) FROM nation GROUP BY 1;"